app/views/pages/accessibility.html.erb

```diff
@@ -2,142 +2,171 @@
+      } %>
+
+      <p class="govuk-body">
+        This accessibility statement applies to Find open data at <%= link_to 'https://data.gov.uk/', '/', class: 'govuk-link' %>
+      </p>
+
+      <p class="govuk-body">
+      This website is run by the GOV.UK team at the Government Digital Service (GDS).
+      We want as many people as possible to be able to use this website. For example, that means you should be able to:
+      </p>
+
+      <%= render "govuk_publishing_components/components/list", {
+        visible_counters: true,
+        items: [
+          "change colours, contrast levels and fonts",
+          "zoom in up to 300% without problems",
+          "navigate most of the website using just a keyboard",
+          "navigate most of the website using speech recognition software",
+          "listen to most of the website using a screen reader (including the most recent versions of JAWS, NVDA and VoiceOver)",
+        ]
+      } %>
+
+      <p class="govuk-body">
+        We’ve also made the website text as simple as possible to understand.
+      </p>
+
+      <p class="govuk-body">
+        <%= link_to 'AbilityNet', 'https://mcmw.abilitynet.org.uk/', class: 'govuk-link' %> has advice on making your device easier to use if you have a disability.
+      </p>
+
+      <%= render "govuk_publishing_components/components/heading", {
+        text: "How accessible this website is",
+        margin_bottom: 4,
+        font_size: "m",
+      } %>
+
+      <p class="govuk-body">
+        We know some parts of this website are not fully accessible for the following reasons:
+      </p>
+
+      <%= render "govuk_publishing_components/components/list", {
+        visible_counters: true,
+        items: [
+          "Find open data links to several external websites belonging to other government organisations. We cannot guarantee that these websites meet our accessibility requirements.",
+          "Some data sets have broken links.",
+        ]
+      } %>
+
+      <%= render "govuk_publishing_components/components/heading", {
+        text: "Feedback and contact information",
+        margin_bottom: 4,
+        font_size: "m",
+      } %>
+
+      <p class="govuk-body">
+        If you need any part of this service in a different format like large print, audio recording or braille, please contact the data set publisher.
+        Their contact details should be on the data set’s page.
+      </p>
+
+      <%= render "govuk_publishing_components/components/heading", {
+        text: "Reporting accessibility problems with this website",
+        margin_bottom: 4,
+        font_size: "m",
+      } %>
+
+      <p class="govuk-body">
+        We’re always looking to improve the accessibility of this website.
+        If you find any problems not listed on this page or think we’re not meeting accessibility requirements, please select ‘Report a problem’ on the data.gov.uk <%= link_to 'Support page', '/support', class: 'govuk-link' %>.
```

Comment:
Should `Support` here be capitalised? 

The link could also include "data.gov.uk" to make it clearer to users that pull out a list of all the links in a page.
```suggestion
        If you find any problems not listed on this page or think we’re not meeting accessibility requirements, please select ‘Report a problem’ on the <%= link_to 'data.gov.uk support page', '/support', class: 'govuk-link' %>.
```